### PR TITLE
Interactive pull request

### PR DESCRIPTION
**Pull request by alicemirigo92**
- [ ] Add event listener to the checkbox
- [ ] Implement a function

### CODE REVIEW

File: src/modules/clear.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 
-//   clearTodos() {
-//     const filtered = this.todos.filter((t) => !t.completed);
-//     localStorage.setItem('todos', JSON.stringify(filtered));
-//   }
-// }
+  clearTodos() {
+    const filtered = this.todos.filter((t) => !t.completed);
+    localStorage.setItem('todos', JSON.stringify(filtered));
+  }
```
Comment (codepantha) [inline src/modules/clear.js]: - [ ] The indexes of the todos are not correctly updated after clearing the completed to-do items. Please implement this change as it messes up the order of the items.
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @alicemirigo92 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Update to-do completed status 📌 
✔️ Clear completed tasks 📌 
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢 🟢 🟢

Hi @alicemirigo92,

Good job working on the changes asked by the previous reviewer 👍🏽

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:
✅ You used Gitflow correctly 👍🏽 
✅ Linter checks passed 👍🏽
✅ Todo list is interactive as expected 👍🏽
✅ Todo items are indexed correctly 👍🏽

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/modules/check.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+  }
 
-//   checkItem() {
-//     const checked = this.todos.find((t) => t.index === +this.id);
-//     if (checked) {
-//       checked.completed = !checked.completed;
-//     }
-//     localStorage.setItem('todos', JSON.stringify(this.todos));
-//   }
-// }
```
Comment (Divinity-dev) [inline src/modules/check.js:14]: This part of your code should be removed to make a cleaner code

File: src/modules/clear.js [hunk 3]
```diff
@@ -1,10 +1,15 @@
-// export default class Clear {
-//   constructor() {
-//     this.todos = JSON.parse(localStorage.getItem('todos')) || [];
-//   }
```
Comment (Divinity-dev) [inline src/modules/clear.js:4]: remove this part too

File: src/modules/clear.js [hunk 4]
```diff
@@ -1,10 +1,15 @@
...
+export default class Clear {
+  constructor() {
+    this.todos = JSON.parse(localStorage.getItem('todos')) || [];
+  }
 
-//   clearTodos() {
-//     const filtered = this.todos.filter((t) => !t.completed);
-//     localStorage.setItem('todos', JSON.stringify(filtered));
-//   }
-// }
```
Comment (Divinity-dev) [inline src/modules/clear.js:10]: Remove this part too

**Review by Divinity-dev — COMMENTED** [review]
Great work you've put into this project so far, just a little change. All commented out codes should be removed to make your code neater.

**Pull request merged**